app/api/ApiServer.js

```diff
@@ -0,0 +1,27 @@
+//This creates new axios instance that is only used by router for server-side api calls.
+//If user is authenticated we create an interceptor for all requests made from server to server that were instigated by client
+//It's basically only used for initial server side rendering
+import axios from 'axios';
+
+let clientConfig = {
+  host: process.env.HOSTNAME || '0.0.0.0',
+  port: process.env.PORT || '8080'
+};
```

Comment:
I think we might need to abstract this to a `clientConfig` file given that we're using it everywhere, but that should be done in a separate PR.